core/src/test/java/org/apache/iceberg/rest/auth/TestBasicAuthManager.java

```diff
@@ -30,7 +30,7 @@ class TestBasicAuthManager {
   @Test
   void missingUsername() {
     try (AuthManager authManager = new BasicAuthManager("test")) {
-      assertThatThrownBy(() -> authManager.catalogSession(null, Map.of()))
+      assertThatThrownBy(() -> authManager.authSession(ImmutableAuthScopes.Standalone.of(Map.of())))
```

Comment:
so from how I understand it, the `Standalone` scope is basically the same as the `Catalog` scope and only differs in naming?

Reply:
Mostly, yes. I tried to find a name that would capture its "generic" purpose better than just calling it `Catalog` or `Table`.